Simplified query interface returns data serialization

wu-clan opened the pull request:
PLR.

downdawn commented:
Why do we need to change it like this? It will be missing the verification of pydantic.

wu-clan commented:
Yes, you're right, it was an unintentional change intended to simplify the operation, I ignored the pydantic verification

wu-clan commented:
Hi, @downdawn 
I reverted to query data validation, the main changes occurring in that the serialization functions are now asynchronous and renamed so that they are now better understood
